features/docs/formatters/message.feature

```diff
@@ -43,3 +43,28 @@ Feature: Message output formatter
       testRunFinished
       """
 
+  Scenario: it sets "testRunFinished"."success" to false if something failed
+    Given a file named "features/steps.rb" with:
+      """
+      Given('a passed step') {}
+      Given('a failed step') { fail }
+      """
+    When I run `cucumber features/my_feature.feature --format message`
+    Then output should be valid NDJSON
+    And the output should contain:
+    """
+    {"testRunFinished":{"success":false
+    """
+
+  Scenario: it sets "testRunFinished"."success" to true if nothing failed
+    Given a file named "features/steps.rb" with:
+      """
+      Given('a passed step') {}
+      Given('a failed step') { skip_this_scenario }
+      """
+    When I run `cucumber features/my_feature.feature --format message`
+    Then output should be valid NDJSON
+    And the output should contain:
+    """
+    {"testRunFinished":{"success":true
```

Comment:
Qu) Is the state only true/false? so pending is classed as true? Could we maybe just make the failed step actually be a passing step just to make me feel better

Reply:
Success is true or false, yes.
I took the same way the CLI use to exit with 0 or not.
Pending step are passing or failing depending the strict mode.

I can change the test as requested, yes 👌